src/auth/getCsrfToken.js

```diff
@@ -46,4 +46,9 @@ const getCsrfToken = async (url, csrfTokenApiPath) => {
   return csrfTokenRequestPromises[domain];
 };
 
+const clearCsrfTokenCache = () => {
```

Comment:
Added to facilitate testing. We needed babel rewire to simulate different states before.